o2a/utils/el_utils.py

```diff
@@ -182,8 +189,9 @@ def comma_separated_string_to_list(line: str) -> Union[List[str], str]:
     return values[0] if len(values) <= 1 else values
 
 
-def normalize_path(url, params, allow_no_schema=False):
-    url_with_var = replace_el_with_var(url, params=params, quote=False)
+def normalize_path(url, props: PropertySet, allow_no_schema=False):
+    url_with_var = replace_el_with_var(url, props=props, quote=False)
+    url_with_var = replace_el_with_var(url_with_var, props=props, quote=False)
```

Comment:
I do not understand the purpose of this change.

Reply:
That was bad merge : fixed in 4a956b1daef8318aea49d8561a77cc04a22e4c4b